### ISSUE TITLE
Sync operation against TFS 2015 Update 2 fails unexpectedly.

### ISSUE THREAD

**Issue by rroman81** [Microsoft/vsts-vscode#223]
I have a VSCode v1.12.1 running against the TFVC local workspace VS2015 Update2.  When attempting to get latest via "Sync" operation I get the following in the output console:
```` cmd
tf status -noprompt -collection:http://<tfserver>:8080/tfs/SoftwareCollection ******** -format:xml -recursive
tf get -noprompt -collection:http://<tfserver>:8080/tfs/SoftwareCollection ******** -nosummary c:\tfs\sandbox\rrozinov\FabricatorPOC -recursive
[Sync] Execution of the TFVC command line failed unexpectedly.
[Sync]  c:\tfs\sandbox\rrozinov\FabricatorPOC> get -noprompt -collection:http://<tfserver>:8080/tfs/SoftwareCollection -login:<login,password> -nosummary c:\tfs\sandbox\rrozinov\FabricatorPOC -recursive
````

**Comment by jeffyoung** [comment]
Hi @rroman81, can you provide a bit more information from the log?

I'd like to know what version of the extension you're running and what version of the TF tools you're using. Since you're on Windows, I suspect that you're using `tf.exe`. If so, can you run `tf.exe add -noprompt -?` and provide the version of the exe here? Lastly, can you drop to the command prompt (in the same folder that you're running from within Visual Studio Code) and run that `tf get` command. I'm interested in understanding what the output is (and why we can't parse it).

**Comment by rroman81** [comment]
Hey @jeffyoung ,
pardon for the delay in my response.

The "tf.exe add -noprompt -?" command returns Version 15.112.26421.0 of the tooling.

When taking the "tf get ..." command from the VSCode and running under command line, it does return an error "Unrecognized command option 'collection'" ... I was able to make the command line succeed by being explicit that i am working with TFVC via this approach:
"tf vc get -noprompt ******** -nosummary <pathtotfvc> -recursive"

I wont have much time during the week, but i can probably carve some time on Sunday to see if I can help with the path via vs2017 tf command.

**Comment by jeffyoung** [comment]
Thanks @rroman81 for the details (and to @nigelgrannum on #224 which appears to be the same issue).

Unfortunately, for me to debug this I'm going to need one of you (or both) to [enable logging](https://github.com/Microsoft/vsts-vscode#logging) and provide the output log.  The extension should _not_ be adding the collection parameter to quite a few of the commands run by TF.exe (and for those commands, the collection parameter is only needed for the TEE CLC). The `get` command is one of these. So if you could turn on debug logging and then reproduce the issue, I'd like to see what else is in the log that might help me figure out what's going on.  Ideally, you would run a few other TFVC commands as well (e.g., if you delete a file you know is under version control, we should detect the delete and mark it as such; the delete command for TF.exe also does not expect the collection parameter and I wonder if that's working as expected).

In one of my dev environments I've got TF.exe v15.112.26410.0 and my `get` command comes back like this:
`tf get -noprompt ******** -nosummary c:\repos\@java-tfs2015\multipleprojects-multiplemappings\DeepSpace -recursive`

While my `status` command comes back like this:
`tf status -noprompt -collection:http://java-tfs2015:8081/tfs/DefaultCollection ******** -format:detailed -recursive`

So this is indeed puzzling. 😕

**Comment by jeffyoung** [comment]
Based on what I discovered with #224, check to see if you have `TF.EXE` instead of `TF.exe` in your settings (notice the casing of the file extension).  The detection logic we use for the executable isn't case-sensitive (yet). A fix is coming.

**Comment by jeffyoung** [comment]
The fix for the case-sensitive file extension was released in v1.118.1 (just a few minutes ago).  Hopefully that was the issue here @rroman81.

**Comment by rroman81** [comment]
I got the v1.118.2 and it looks like the issue is fixed now.  Thank you.
